dispatcher/MongoConnect.py

```diff
@@ -293,8 +260,28 @@ def aggregate_status(self):
         except Exception as e:
             self.log.error(f'DB snafu? Couldn\'t update aggregate status. '
                             f'{type(e)}, {e}')
+
+        self.physcal_status = phys_stat
```

Comment:
Is this used somewhere, at least I cannot find it?

Reply:
This is only used by the hypervisor. It's also misspelled.